pkg/drivers/kic/oci/network_create.go

```diff
@@ -136,12 +138,16 @@ type netInfo struct {
 }
 
 // if exists returns subnet, gateway and mtu
-func dockerNetworkInspect(name string) (netInfo, error) {
+func dockerNetworkInspect(ociBin string, name string) (netInfo, error) {
```

Comment:
does this need to be renamed to networkInspect ? if it is working for both podman and docker ?

Reply:
There is "docker" all over the place, I didn't rename any functions.

Reply:
I mean if we are using it for both docker and podman now (unlike before) we should rename it to networkInspect